### ISSUE TITLE
Color contrast fail under AA WCAG standards (LibreTranslate website)

### ISSUE THREAD

**Issue by idotj** [LibreTranslate/LibreTranslate#197]
Checking the colors with a tool to validate the AA level WCAG will fail because the contrast doesn't 

After checking the website with a WCAG Color contrast validator, I saw that the recommended level (AA) fails because of the low contrast between the light blue and white:

![image](https://user-images.githubusercontent.com/6554216/149819115-2b7ede65-e14f-4928-9357-205ab9999a8f.png)

A change of colors will fix the problem.
